pkgs/development/python-modules/webargs/5.nix

```diff
@@ -0,0 +1,24 @@
+{ lib, buildPythonPackage, fetchPypi
+, marshmallow, pytest, webtest
+}:
+
+buildPythonPackage rec {
+  pname = "webargs";
+  version = "5.5.3";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "16pjzc265yx579ijz5scffyfd1vsmi87fdcgnzaj2by6w2i445l7";
+  };
+
+  buildInputs = [ pytest webtest ];
+  propagatedBuildInputs = [ marshmallow ];
+
+  doCheck = false;
```

Comment:
Please add a comment why tests are disabled and add a pythonImportsCheck.

Reply:
Try pytestCheckHook.

Reply:
There are no tests in the pypi archive. Should we get them from GitHub or is that a good enough reason to set `doCheck = false`?

Reply:
I hope it's a good reason. I added that.

Reply:
It is pretty standard to fetch from GitHub instead.